docs/qna.md

```diff
@@ -9,14 +9,12 @@ OpenAPI generator (openapi-generator) is managed by the [core team members](../R
 
 There are several reasons:
 
-1. The founding members came to the conclusion that Swagger Codegen 3.0.0 beta contains too many breaking changes while they strongly believe 3.0.0 release should only focus on one thing: OpenAPI specification 3.0 support.
-1. Swagger Codegen 3.0.0 beta was evaluated as unstable. Changes made directly to 3.0.0 branch without reviews or tests, were breaking the builds from time to time (e.g. a simple `mvn clean package` failed).
-1. Reviews of code changes in the 3.0.0 branch highlighted a lot of code block removal without any reason. This might produce regressions for edge cases discovered previously.
-1. Most of the test cases in the generators have been commented out as part of the migration to support OpenAPI 3.0. Test cases are the most valuable assets of the project and should be maintained to ensure a good quality.
-1. According to SmartBear, [Swagger Codegen 2.x and 3.x should be supported in parallel for a while](https://github.com/swagger-api/swagger-codegen/issues/7754#issuecomment-375039048) without the possibility to work with git branches to merge the fixes from one branch to the next. Having to implement everything twice is not a good idea and the best use of the Swagger Codegen community resources.
-1. Having a community-driven version can bring the project to the next level.
-
-UPDATE: After the public launch of OpenAPI Generator, some of the core team members were contacted by Ron (SmartBear) and we explained to him the reasons of the fork as explained above. We also asked him to contact team@openapitools.org if he wants to discuss further but so far we don't see any email from him to team@openapitools.org.
+1. The founding members came to the conclusion that Swagger Codegen 3.0.0 beta contains too many breaking changes while they strongly believe 3.0.0 release should only focus on one thing: OpenAPI 3.0 support.
+1. We had concerns about the development practices, which seemed to be contributing to an unstable and insufficiently tested codebase.
+1. We could not agree on an efficient evolutionary strategy.  The founding members felt it was important to move forward with OpenAPI 3.0 support, while maintaining backward compatibility with OpenAPI 2.0 in the same codebase.
+1. We found that the enhancements, bug fixes submitted for Swagger Codegen 2.x need to be submitted again for Swagger Codegen 3.0.0 branch. Otherwise, these changes will not appear in the 3.0.0 branch. Having to do the PR twice is not the best use of the community resource.
```

Comment:
Isn't this just a consequence of the previous point? If so, consider leaving it out.  

If you think the consequence is important, maybe just append to the previous point:  "...in the same codebase, rather than maintaining two separate branches in parallel."

Reply:
Thanks for the feedback. It may or may not be the consequence of the previous point and that's why I think we should elaborate with a separate point.

In OpenAPI Generator, we maintain different branches for patch, minor and major (e.g. 4.0.x with breaking chagnes) releases and we regularly sync changes between these branches to reduce community effort in submitting duplicated PRs.

In other words, maintaining different branches for the project to evolve is perfectly fine but not having a practice/workflow to sync enhancements, bug fixes between branches is less than ideal.

(These are my opinions)